archivers/unshield/Portfile

```diff
@@ -12,25 +13,23 @@ long_description \
 	Extract files from InstallShield-style cabinet files (usually named \
 	data1.cab, data1.hdr, data2.cab, etc), versions 5 and later
 
+homepage			https://github.com/twogood/unshield
+master_sites		https://github.com/twogood/unshield/archive/refs/tags/
+distfiles           ${version}${extract.suffix}
 
-homepage			http://synce.sourceforge.net/synce/unshield.php
-master_sites		sourceforge:synce
+depends_lib			port:libiconv \
+                    port:zlib
 
-depends_lib			port:zlib
+checksums			md5     ba0759866923a62be5ca799232c2b480 \
+					sha256  aa8c978dc0eb1158d266eaddcd1852d6d71620ddfc82807fe4bf2e19022b7bab \
+					rmd160  0f26f1c0d3637762fa06a3c2d60253046f0ed085 \
+                    size    58822
 
-checksums			md5     31a829192a255160d1f71cda4c865c9c \
-					sha1    3e1197116145405f786709608a5a636a19f4f3e1 \
-					rmd160  1370621e38736f14dab1ec540b9233906b5eaa31
-
-patchfiles			patch-src_Makefile.in.diff
-
-configure.args		--with-zlib=${prefix}
+configure.ldflags-append -liconv
```

Comment:
this doesn't look like the typical CMake based way of doing things...

Reply:
I'm not a CMake expert, but it wouldn't build without this. There may be a better way, but I don't know what it is. I found this by searching through all the existing portfiles for Cmake and iconv, and seeing how other people did it, and copying to see what worked.

Reply:
Also, is there a preference for tabs vs spaces?

Reply:
Spaces please

Reply:
the preference for spaces is part of the modeline; if you check the port `lint` output of the CI (or locally) you see a bunch of warnings related to this, which should be fixed.

Reply:
you're right about the `configure.ldflags-append -liconv` it doesn't build without it, so leave it

Reply:
Fixed the whitespace.